datanommer.models/alembic/versions/1d4feffd78fe_add_historic_user_an.py

```diff
@@ -83,15 +83,15 @@ def upgrade():
 
         if not msg.users:
             new_usernames = msg2usernames(msg.__json__(), **config)
-            print "Updating users to %r" % new_usernames
+            print("Updating users to %r" % new_usernames)
```

Comment:
Well I thought the same and then https://github.com/jmflinuxtx/kerneltest-harness/pull/21/files#r48329864 since, I changed my mind :)